### DISCUSSION TITLE
Finetuning SigLiT/ SigLIP details, optimizers, hyperparams, learning rate schedulers.

### DISCUSSION THREAD

**Discussion by sharadchandakacherla** [google-research/big_vision#128] [General]
Has anyone tried finetuning SigLiT/ SigLIP like model on a particular task, I am facing issues with very low loss values, and suspect the gradients not to be moving at all. If some one has tried it, would like to learn from their experiences.
